### ISSUE TITLE
Attribute autocomplete missing quotes

### ISSUE THREAD

**Issue by patrycjanie** [PAYONE-GmbH/magento-2#585]
Hi, 
during A11y improvements in Magento 2 project, I noticed serious issue related to autocomplete. Many of the templates located in `view/frontend/web/template/payment` are missing quotes when assigning autocomplete attribute. Example: 
**https://github.com/PAYONE-GmbH/magento-2/blob/c5ed1c741d9e776c215105c1e4be8939f7dad6cc/view/frontend/web/template/payment/creditcard.html#L103**

This results in `off` method `vendor/magento/module-ui/view/base/web/js/lib/core/events.js` being assigned instead of assigning value `off` to this attribute.

```
<input type="text" name="payment[cc_cardholder]" class="input-text" value="" data-bind="attr: {
                                        autocomplete: off,
                                        id: getCode() + '_cardholder',
                                        title: $t('Cardholder'),
                                        'data-container': getCode() + '-cardholder'},
                                  value: cardholder,
                                  valueUpdate: 'keyup' " autocomplete="function (ns) {
            var storage = getEvents(this);

            if (!storage) {
                return this;
            }

            storage.forEach(function (handlers, name) {
                handlers = handlers.filter(function (handler) {
                    return !ns ? false : handler.ns !== ns;
                });

                handlers.length ?
                    storage.set(name, handlers) :
                    storage.delete(name);
            });

            return this;
        }" id="payone_creditcard_cardholder" title="Karteninhaber" data-container="payone_creditcard-cardholder">
```

Can you please address this issue as this is flagged as error during a11y audits

<img width="1182" alt="Image" src="https://github.com/user-attachments/assets/1c3318d4-b2e3-4a8e-8635-70b62f7283b9" />

**Comment by janteuber** [comment]
Hello @patrycjanie 

Thank you for your feedback! We have fixed the issue immediately. If you have any further questions or concerns, please feel free to let us know!

**Comment by patrycjanie** [comment]
thanks for your quick response!
